workspaces/marketplace/packages/cli/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@red-hat-developer-hub/marketplace-cli",
+  "description": "CLI for the Marketplace",
+  "version": "0.0.1",
```

Change this to 0.0.0 so that the release process will build the 0.0.1.

For this please add a changeset.

```suggestion
  "version": "0.0.0",
```